angular/projects/catalyst/src/lib/catalyst.module.ts

```diff
@@ -5,7 +5,11 @@ import * as Components from './directives/proxies';
 const CatComponents = [
   Components.CatButton,
   Components.CatIcon,
-  Components.CatSpinner
+  Components.CatSpinner,
```

Comment:
Was this autogenerated?

Reply:
No, I added them to include the new components that we added.

Reply:
ah, makes sense. i thought the stencil CLI already takes care of it but apparently it does not :D